### ISSUE TITLE
Demo does not work on chromium 44.0.2403.157 (64-bit) in Linux VM

### ISSUE THREAD

**Issue by joIivier** [gaearon/react-dnd#273]
I tried the chessboard demo at http://gaearon.github.io/react-dnd/examples-chessboard-tutorial-app.html and the drop part does not work on my environment: I can pick the horse, the cells where I can put it are highlighted, the horse head image appears and follows my cursor but I don't see the cell turning red or green when I hover them and nothing happens when I drop the horse, it gets back to where I picked it.
What is interesting is that this demo **works on**:
- Google Chrome 44.0.2403.157 m on Windows
- Chromium 47.0.2398.0 on Windows
- Firefox 40.0.3 on Linux or Windows

But **doesnt work on**:
- Google Chrome Version 44.0.2403.157 (64-bit) on Linux
- Chromium 46.0.2471.2 on Linux

So since my Linux is in a VM and there were recently other bugs related to the mouse handling in chrome/chromium running in VirtualBox this might be a bug in chromium instead of your library. What can be done to investigate this?

Thanks

Olivier

**Comment by gaearon** [comment]
Try HTML5 drag and drop examples like [this one](http://html5demos.com/drag).

**Comment by joIivier** [comment]
Hi, this demo works on all the browsers I tested above, especially the chrom(ium|e) on Linux, so this leads to an issue in react-dnd instead of the browser. What is the next troubleshooting step?

**Comment by gaearon** [comment]
Try every other demo from the website and see which ones work and which don't.

**Comment by joIivier** [comment]
Ok so they all work, the last one (http://html5demos.com/drag-anything) seems to be a little bit broken on all the browsers I tested (we see JS codes in the UI, the image link is broken). The only difference I can notice is that I cannot drag the image from this demo on chromium on linux but I can do it on firefox on linux; but this might be due to the fact that the image is not found. And I am not sure this is linked to the initial issue since the initial issue is about the drop not doing anything, here this is the drag that doesn't work.
And all other demos everything is fine.

**Comment by gaearon** [comment]
Whoops, I'm so sorry, I meant other demos from React DnD website: 

http://gaearon.github.io/react-dnd/examples-dustbin-single-target.html
http://gaearon.github.io/react-dnd/examples-dustbin-multiple-targets.html

and so on (in the sidebar)

**Comment by joIivier** [comment]
Oh ok, so by testing again I discovered the root of the problem: all the DnD demos (from html5demos or your examples) work fine on my primary screen on all browsers and don't work on chromium/chrome on my secondary screen in my linux VM! I happened to have previously tested html5demos on my primary screen and your demos on my secondary.
So this is clearly not an issue with React DND but more with the multi screen integration of my VM and its communication with chromium.
Thanks for your support anyway!
Regards
Olivier
